### PR TITLE
adding updated readme

### PR DESCRIPTION
**Pull request by yanningcui**
Yanning Cui hw01

### CODE REVIEW

**Comment by XianQu2016** [comment]
Coding Skills: Excellent
Coding Strategy: Excellent
Achievement, creativity: Excellent
Ease of Access: Excellent
Remarks: 
Good job! The table you inserted on top seems nice. You also mentioned that the output is PDF. document. While learning from your code, I’m also quite curious about the difference between pdf output and other formats(?). The final file shows that you effectively used five style elements including photo, italics, headers, list, and link. In fact, the sushi picture you inserted is quite creative and seems delicious! Also, I’m wondering how the outcome of your list “1. by region; 2. by gender” isn’t indented with your former list while in the code it is. Besides, header “hw01; Repository for first CFSS homework assignment” might be deleted, and your name could be added when pulling request. All in all, nice job and look forward to your future work!

File: README.md [hunk 1]
```diff
@@ -1,2 +1,38 @@
...
+
+My research interest is computational content analysis using Twitter API.
+
+To be specific,
+
+- Twitter users' attitudes(**emotions** and **sentiments**) toward a specific event
+- Patterns of attitudes over an amount of time:
+   1.by region(the user's location)
+   2.by gender
+
```
Comment (XianQu2016) [inline README.md:22]: This seems great! I’m wondering how the outcome of your list “1. by region; 2. by gender” isn’t indented with your former list while in the code it is.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,38 @@
...
+##Enough academics!
+> I also love cooking and enjoy music!
+
+Figure 1 is the image of sashimi.
+
+![sashimi](https://s-media-cache-ak0.pinimg.com/736x/c8/c7/27/c8c7270cc815d8967bec0174b4aa7e76.jpg)
+
+You can [link to Wikipedia!](https://en.wikipedia.org/wiki/Sashimi)
+
```
Comment (XianQu2016) [inline README.md:31]: the sushi picture you inserted is quite creative and seems delicious!

File: README.md [hunk 3]
```diff
@@ -1,2 +1,38 @@
...
+##Description
+I followed this [tutorial](https://guides.github.com/features/mastering-markdown/) to use markdown and make essential changes to readme.md.It took me a while to download MacTex (for exporting pdf R markdown files for future use), and restart R, and open a R Markdown file.
+I tried both markdown(.md) and R markdown(.Rmd), and results are the same.I also debug my scripts, and preview pdf/knit pdf to preview the result.
+
+I tried this [tutorial](https://rstudio.github.io/dygraphs/r-markdown.html) to resize the image, but still couldn't get it inserted right after the text in the perfect size. 
+
+Because I have used GitHub before, in another class last quarter, so I am quite familiar with git clone, commit, push, etc.
```
Comment (XianQu2016) [inline README.md:38]: The final file shows that you effectively used five style elements including photo, italics, headers, list, and link.All in all, nice job and look forward to your future work!

File: README.md [hunk 4]
```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
 # hw01
 Repository for first CFSS homework assignment
```
Comment (XianQu2016) [inline README.md:8]: header “hw01; Repository for first CFSS homework assignment” might be deleted, and your name could be added when pulling request.

File: README.md [hunk 5]
```diff
@@ -1,2 +1,38 @@
+---
+title: "CFSS hw01"
+author: "Yanning Cui"
+date: "January 9, 2017"
+output: pdf_document
+---
```
Comment (XianQu2016) [inline README.md:6]: The table you inserted on top seems nice. You also mentioned that the output is PDF. document. While learning from your code, I’m also quite curious about the difference between pdf output and other formats(?).

**Review by XianQu2016 — APPROVED** [review]
Overall nice job! (This is a complement to my last comment following review instruction)

File: README.md [hunk 6]
```diff
@@ -1,2 +1,38 @@
...
+
+![sashimi](https://s-media-cache-ak0.pinimg.com/736x/c8/c7/27/c8c7270cc815d8967bec0174b4aa7e76.jpg)
+
+You can [link to Wikipedia!](https://en.wikipedia.org/wiki/Sashimi)
+
+##Description
+I followed this [tutorial](https://guides.github.com/features/mastering-markdown/) to use markdown and make essential changes to readme.md.It took me a while to download MacTex (for exporting pdf R markdown files for future use), and restart R, and open a R Markdown file.
+I tried both markdown(.md) and R markdown(.Rmd), and results are the same.I also debug my scripts, and preview pdf/knit pdf to preview the result.
+
+I tried this [tutorial](https://rstudio.github.io/dygraphs/r-markdown.html) to resize the image, but still couldn't get it inserted right after the text in the perfect size.
```
Comment (limchengyee) [inline README.md:36]: I also attempted to change the size of the image embedded through the markdown document, but was unable to do so. But, when googling, I found out that I could change the image size by using html code.

File: README.md (hunk 5 above)
Comment (limchengyee) [reply README.md:6]: The table was neat and comprehensive. I think it'd be helpful for other students if you included the pdf output from your code in your repository as well.

**Review by limchengyee — COMMENTED** [review]
Evaluation of HW01:

Coding Style: +
Coding Strategy: +
Presentation: + (Clear and comprehensive)
Achievement, Creativity: +
Ease of Access: Excellent +

Reviews:
Good job! This assignment is aesthetically pleasing, clear and concise. It also demonstrated her competence in Rmarkdown (i.e. bold, italics, hyperlinks, pictures, lists and headers), even though she had one bug on her ordered list after the unordered list.

One thing that could have been improved in the workflow will be multiple saving, committing and pushing to GitHub as she fixed her biography, so that she could always go back to her previous "checkpoint" in her code (which would be more important when the code in the assignment becomes more complicated. Great job with your first attempt in RMarkdown.

File: README.md (hunk 5 above)
Comment (bensoltoff) [reply README.md:6]: PDF output is one option for R Markdown documents. Though in this context it actually is unnecessary. Markdown documents are automatically rendered on GitHub.com, so you only need to output it to PDF or HTML if it is to be viewed elsewhere. We'll see this soon enough on hw02

File: README.md (hunk 1 above)
Comment (bensoltoff) [reply README.md:22]: You need to indent the numeric lines and add spaces between the periods and the text. Like this:

```
- Twitter users' attitudes(**emotions** and **sentiments**) toward a specific event
- Patterns of attitudes over an amount of time:
    1. by region(the user's location)
    1. by gender
```

- Twitter users' attitudes(**emotions** and **sentiments**) toward a specific event
- Patterns of attitudes over an amount of time:
    1. by region(the user's location)
    1. by gender

Note that you also don't have to number all the bullet points yourself. Each can be "1." and GitHub/R Markdown does the counting for you

File: README.md (hunk 6 above)
Comment (bensoltoff) [reply README.md:36]: Yeah, Markdown doesn't support this. Either manually adjust the picture size on your computer or use HTML code to change it.

**Review by bensoltoff — COMMENTED** [review]
Evaluation HW-01 for Yanning Cui
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |   x           |            |
| **Coding strategy**         |           |     x         |            |
| **Presentation**            |    x       |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Remarks:

* Hit all the marks and utilized the requested number of features
* What if you wanted to use an image saved on your computer? How could you add that to the readme file? Aka, how do you use images that are stored in the repo itself?
* Not a significant commit history. Make sure to commit early and often. It doesn't mean you have to commit every 5 minutes, but especially for larger assignments/projects you should commit files whenever you've made a significant change or updated a major chunk of code.

**Pull request closed without merging**